mergify_engine/tests/unit/test_json_enum.py

```diff
@@ -0,0 +1,38 @@
+# -*- encoding: utf-8 -*-
+#
+# Copyright © 2018–2020 Mergify SAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+import json
+
+from mergify_engine import json_enum
+
+
+with_enum = {
+    "name": "hello",
+    "conditions": [],
+    "actions": {"merge": {"strict": json_enum.StrictMergeParameter.fasttrack}},
+}
+
+
+def test_strict_merge_parameter_encoder():
+    json_file = json.dumps(with_enum, cls=json_enum.StrictMergeParameterEncoder)
+    assert isinstance(json_file, str) is True
```

Comment:
This actually test `json.dumps`, which *always* returns a `str`. So you're not really testing your code but more `json.dumps`.
I think you can just get rid of this test TBH.